packages/vue/src/view/article/ArticleLink.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <a v-bind:href="articleUrl" v-bind:class="className">
+    <slot />
+  </a>
+</template>
+
+<script>
+  export default {
+    props: ['article', 'className'],
```

Comment:
is advised define the type props like this:

```js
props: {
    article: {
       type: string,
       required: true
    }
}
```

https://br.vuejs.org/v2/guide/components-props.html#Prop-Validation